tee/params.go

```diff
@@ -51,7 +51,7 @@ func (p Parameters) EpochStartBlock(epoch uint64) uint64 {
 	return p.InitBlock + epoch*p.PhaseDuration
 }
 
-func (p Parameters) DepositEndBlock(epoch uint64) uint64 {
+func (p Parameters) EpochEndBlock(epoch uint64) uint64 {
 	return p.EpochStartBlock(epoch) + p.PhaseDuration
 }
 
```

Comment:
The naming of those last 3 methods is imprecise.
- `EpochStartBlock` should rather be `DepositStartBlock`
- `EpochEndBlock` should rather be `DepositDoneBlock` or `TxStartBlock`, depending on usage
- `TxEndBlock` should rather be `TxDoneBlock` or `ExitStartBlock`, depending on usage